content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,201 @@
 :show-content:
-:hide-page-toc:
 
 ====================
 Marketing Automation
 ====================
 
-The Odoo *Marketing Automation* app automates a variety of marketing tasks by combining specific
-rules and filters to generate timed actions. Instead of manually having to build each stage of a
-campaign (like a series of timed massmails), the *Marketing Automation* app allows marketers to
-build the entire campaign, and all of its stages, in one place---on one dashboard.
+The Odoo *Marketing Automation* application enables users to create dynamic campaigns with actions
+that automatically occur within a defined duration, such as sending a series of timed mass emails or
+engaging with leads based on their interactions with marketing campaigns.
+
+While the application is designed to be user-friendly for quickly creating, launching, and reviewing
+marketing campaigns, it also provides advanced features to automate repetitive tasks throughout the
+database.
+
+Get started by creating a :ref:`new campaign from scratch <marketing_automation/campaigns>` or start
+with a :ref:`campaign template <marketing-automation/campaign-templates>`.
 
 .. seealso::
    `Odoo Tutorials: Marketing <https://www.odoo.com/slides/marketing-27>`_
 
-Campaign configuration
-======================
+.. cards::
 
-To create a new automated marketing campaign, navigate to :menuselection:`Marketing Automation app
---> New` to reveal a blank campaign form.
+   .. card:: Targeting an audience
```

Comment:
```suggestion
   .. card:: Target an audience
```

Reply:
tbh we should rename this probably to "Audience targeting" or something similar.

Reply:
Agreed, I will make this adjustment in a future PR for [IMP] the `target_audience.rst` doc!